e2e/content-services/directives/create-library-directive.e2e.ts

```diff
@@ -69,9 +69,8 @@ describe('Create library directive', function () {
         done();
     });
 
-    afterEach(async (done) => {
-        await browser.actions().sendKeys(Key.ESCAPE).perform();
-        done();
+    afterEach(() => {
+        browser.refresh();
```

Comment:
maybe BrowserActions.closeMenuAndDialogs(); would be enough here instead of a browser.refresh ...